src/CanCommunity.ts

```diff
@@ -149,52 +164,235 @@ export class CanCommunity {
     return this.signTrx(trx);
   }
 
-  async createCode(input: Createcode, execCodeInput?: ExecCodeInput): Promise<any> {
-    const packedParams = await serializeActionData(this.config, ActionNameEnum.CREATECODE, input);
+  async createSetRightHolderForCodeActionInput(
```

Comment:
i think this func not only set right holder, it includes set type, right holder, rule... so i think should rename to createExecutionRightForCodeActionInput. @danielAlvess

Reply:
updated